src/runtime/rpc/asio_net_provider.cpp

```diff
@@ -37,14 +37,27 @@ asio_network_provider::asio_network_provider(rpc_engine *srv, network *inner_pro
     : connection_oriented_network(srv, inner_provider)
 {
     _acceptor = nullptr;
+    _service_count =
+        (int)dsn_config_get_value_uint64("network",
+                                         "io_service_worker_count",
+                                         1,
+                                         "thread number for io service (timer and boost network)");
+
+    for (std::size_t i = 0; i < _service_count; i++) {
+        io_context_ptr io_context(new boost::asio::io_context(1));
```

Comment:
What's the meaning of `1` in `new boost::asio::io_context(1)`

Reply:
It means the io_context runs in the single thread environment
Ref: https://stackoverflow.com/questions/40829143/what-does-concurrency-hint-of-boostasioio-service-means